main/settings_course_etl.py

```diff
@@ -57,13 +57,8 @@
 MITX_ONLINE_PROGRAMS_API_URL = get_string("MITX_ONLINE_PROGRAMS_API_URL", None)
 MITX_ONLINE_COURSES_API_URL = get_string("MITX_ONLINE_COURSES_API_URL", None)
 
+
 # Open Learning Library settings
-OLL_API_URL = get_string("OLL_API_URL", None)
-OLL_API_ACCESS_TOKEN_URL = get_string("OLL_API_ACCESS_TOKEN_URL", None)
```

Comment:
we should remove other references to these settings. There are some in env/codespaces.env and one in learning_resources/etl/conftest.py

Reply:
Good catch, removed